### ISSUE TITLE
UI/UX design doc discussion

### ISSUE THREAD

**Issue by tianhaoz95** [tianhaoz95/acumany-re#40]
https://tianhaoz.com/acumany-re/#/design-docs/ui-ux-study 

 Description

**Comment by ninjatron** [comment]
We need to start discussing initial features. We can post thoughts as they come.

**Comment by ninjatron** [comment]
I think for beginning we should target following features:

- Ability to set up meeting,
- Send someone meeting invitation,
- Ability to let multiple people join a meeting,
- Video, text chat,
- Whiteboard,

Let me know if anything is missing/doesn't make sense initially.

**Comment by tianhaoz95** [comment]
> I think for beginning we should target following features:
> 
> * Ability to set up meeting,
> * Send someone meeting invitation,
> * Ability to let multiple people join a meeting,
> * Video, text chat,
> * Whiteboard,
> 
> Let me know if anything is missing/doesn't make sense initially.

Yes, agree. To me, I will probably further prioritize it in descending the order you listed where the ability to set up a meeting is the first to build.

@ninjatron the application is checked in. I can add more docs later. Basically, now it is a barebone app, but with some infra to test and deploy. By the end of this week, I will add a few high-level structures so that we can start adding some UI.
